src/scvmm/azext_scvmm/commands.py

```diff
@@ -0,0 +1,137 @@
+
+    with self.command_group('scvmm vmmserver', client_factory=cf_vmmserver) as g:
+        g.custom_command('connect', 'connect_vmmserver', supports_no_wait=True)
+        g.custom_command('update', 'update_vmmserver', supports_no_wait=True)
+        g.custom_command('delete', 'delete_vmmserver', supports_no_wait=True)
+        g.custom_show_command('show', 'show_vmmserver')
+        g.custom_command('list', 'list_vmmserver')
+
+    with self.command_group('scvmm cloud', client_factory=cf_cloud) as g:
+        g.custom_command(
+            'create',
+            'create_cloud',
+            supports_no_wait=True,
+            validator=validate_param_combos,
+        )
+        g.custom_command('update', 'update_cloud', supports_no_wait=True)
+        g.custom_command(
+            'delete', 'delete_cloud', supports_no_wait=True, confirmation=True
+        )
+        g.custom_show_command('show', 'show_cloud')
+        g.custom_command('list', 'list_cloud')
+
+    with self.command_group(
+        'scvmm virtual-network', client_factory=cf_virtual_network
+    ) as g:
+        g.custom_command(
+            'create',
+            'create_virtual_network',
+            supports_no_wait=True,
+            validator=validate_param_combos,
+        )
+        g.custom_command('update', 'update_virtual_network', supports_no_wait=True)
+        g.custom_command(
+            'delete', 'delete_virtual_network', supports_no_wait=True, confirmation=True
+        )
+        g.custom_show_command('show', 'show_virtual_network')
+        g.custom_command('list', 'list_virtual_network')
+
+    with self.command_group(
+        'scvmm vm-template', client_factory=cf_virtual_machine_template
+    ) as g:
+        g.custom_command(
+            'create',
+            'create_vm_template',
+            supports_no_wait=True,
+            validator=validate_param_combos,
+        )
+        g.custom_command('update', 'update_vm_template', supports_no_wait=True)
+        g.custom_command(
+            'delete', 'delete_vm_template', supports_no_wait=True, confirmation=True
+        )
+        g.custom_show_command('show', 'show_vm_template')
+        g.custom_command('list', 'list_vm_template')
+
+    with self.command_group('scvmm vm', client_factory=cf_virtual_machine) as g:
+        g.custom_command(
+            'create',
+            'create_vm',
+            supports_no_wait=True,
+            validator=validate_param_combos_for_vm,
+        )
+        g.custom_command(
+            'delete', 'delete_vm', supports_no_wait=True, confirmation=True
+        )
+        g.custom_command('update', 'update_vm', supports_no_wait=True)
+        g.custom_show_command('show', 'show_vm')
+        g.custom_command('list', 'list_vm')
+        g.custom_command('start', 'start_vm', supports_no_wait=True)
+        g.custom_command('stop', 'stop_vm', supports_no_wait=True)
+        g.custom_command('restart', 'restart_vm', supports_no_wait=True)
```

Comment:
Please add "wait" commands for this command group

Reply:
Added wait command